### ISSUE TITLE
twitter.com/author/tweets should be redirected to preferred.nitter.instance/author

### ISSUE THREAD

**Issue by B0pol** [SimonBrazell/privacy-redirect#51]
For example, https://twitter.com/simonbrazell/tweets
It leads to 404 now.

**Comment by SimonBrazell** [comment]
No worries, I'll look into it.

**Comment by SimonBrazell** [comment]
Is the `/author/tweets` path a legacy one or something? Just seems to get redirected to `/author` anyway on Twitter, where are you coming across these links?

**Comment by B0pol** [comment]
I just tried disabling javascript and using legacy twitter. It doesn't redirecs `/author/tweets` to `/author` and will show tweets. But it seems the true legacy path is `/author?p=s`. I mean it works without `?p=s`, but when you click on someone's profile via another tweet, it leads to `/author?p=s`.

I came across this link on reddit. 

Maybe some twitter clients use `/author/tweets` and it never caused problems until now because it works on both old & new twitter?
